src/Apps/CollectorProfile/Routes/Saves/Components/EditListPrivacyModal/EditArtworkListItem.tsx

```diff
@@ -0,0 +1,117 @@
+import { Flex, Image, Toggle, Text, Spacer } from "@artsy/palette"
+import { FC } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { extractNodes } from "Utils/extractNodes"
+import { EditArtworkListItem_item$data } from "__generated__/EditArtworkListItem_item.graphql"
+import { useFormikContext } from "formik"
+import { __internal__useMatchMedia } from "Utils/Hooks/useMatchMedia"
+import NoArtIcon from "@artsy/icons/NoArtIcon"
+import { EditListPrivacyFormModel } from "Apps/CollectorProfile/Routes/Saves/Components/EditListPrivacyModal/EditListPrivacyModal"
+import HideIcon from "@artsy/icons/HideIcon"
+
+interface EditArtworkListItemProps {
+  item: EditArtworkListItem_item$data
+}
+
+const EditArtworkListItem: FC<EditArtworkListItemProps> = props => {
+  const { item } = props
+  const artworkNodes = extractNodes(item.artworksConnection)
+  const imageURL = artworkNodes[0]?.image?.resized?.src ?? null
+  const totalArtworks = item.artworksCount ?? 0
+  const { values, setFieldValue } = useFormikContext<EditListPrivacyFormModel>()
+
+  return (
+    <Flex justifyContent="space-between">
+      <Flex alignItems="center">
+        {imageURL ? (
+          <Image
+            src={imageURL ?? ""}
+            width={[40, 60]}
+            height={[40, 60]}
+            lazyLoad
+            alt="Artwork image"
+          />
+        ) : (
+          <ArtworkImagePlaceholder />
+        )}
+
+        <Flex flexDirection="column">
+          <Flex alignItems="center">
+            <Text variant={["xs", "sm"]} paddingLeft={1}>
+              {item.name}
+            </Text>
+
+            <Spacer x={[0.5, 1]} />
+
+            {!values[item.internalID] && <HideIcon />}
```

Comment:
This one might also need a `minWidth` to avoid this shrinking issue that I saw on the other pages. Forgot to add this in a first place.